features/gba-dev/europees-kiesrecht-gba.feature

```diff
@@ -0,0 +1,62 @@
+    | aanduiding.omschrijving  | <aanduiding.omschrijving> |
+
+    Voorbeelden:
+   | omschrijving                       | Europees kiesrecht | aanduiding.code | aanduiding.omschrijving |
+   | uitgesloten zonder einddatum       | 1                  | 1               | persoon is uitgesloten  |
+   | persoon ontvangt oproep            | 2                  | 2               | persoon ontvangt oproep |
+
+
+  Abstract Scenario: Europees kiesrecht <omschrijving>
+    Gegeven de persoon met burgerservicenummer '000000127' heeft de volgende 'inschrijving' gegevens
+    | Europees kiesrecht (31.10) | einddatum uitsluiting Europees kiesrecht (31.30) |
+    | <Europees kiesrecht>       | <einddatum uitsluiting>                          |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                                |
+    | type                | RaadpleegMetBurgerservicenummer       |
+    | burgerservicenummer | 000000127                             |
+    | fields              | burgerservicenummer,europeesKiesrecht |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                                       | waarde                    |
+    | burgerservicenummer                        | 000000127                 |
+    | europeesKiesrecht.aanduiding.code          | <aanduiding.code>         |
+    | europeesKiesrecht.aanduiding.omschrijving  | <aanduiding.omschrijving> |
+    | europeesKiesrecht.einddatumUitsluiting     | <einddatumUitsluiting>    |
+
+    Voorbeelden:
+   | omschrijving                       | Europees kiesrecht | einddatum uitsluiting | aanduiding.code | aanduiding.omschrijving | einddatumUitsluiting |
+   | einddatum uitsluiting in toekomst  | 1                  | volgend jaar          | 1               | persoon is uitgesloten  | volgend jaar         |
+   | einddatum uitsluiting in verleden  | 1                  | gisteren              |                 |                         |                      |
+   | einddatum uitsluiting vandaag      | 1                  | vandaag               |                 |                         |                      |
+   | einddatum uitsluiting morgen       | 1                  | morgen                | 1               | persoon is uitgesloten  | morgen               |
+   | einddatum uitsluiting vorige maand | 1                  | vorige maand          |                 |                         |                      |
+   | einddatum uitsluiting deze maand   | 1                  | deze maand            | 1               | persoon is uitgesloten  | deze maand           |
+   | einddatum uitsluiting vorig jaar   | 1                  | vorig jaar            |                 |                         |                      |
+   | einddatum uitsluiting dit jaar     | 1                  | dit jaar              | 1               | persoon is uitgesloten  | dit jaar             |
```

Comment:
je bedoelt dit deel van het jaar dat nog komen gaat?

Reply:
Hier wordt het jaar waarin we leven teruggeven. (dus t.m 31 december is dit 2022).